File: UserManagementFront/src/app/shared/models/dtos/user-statistics.dto.ts
export interface UserStatisticsDto {
  statusStats: StatusStatisticsDto;
  roleStats: RoleStatisticsDto;
}

export interface StatusStatisticsDto {
  active: number;
  inactive: number;
  activePercentage: number;
  inactivePercentage: number;
}

export interface RoleStatisticsDto {
  admin: number;
  user: number;
  adminPercentage: number;
  userPercentage: number;
}

export interface StatisticsResponse {
  success: boolean;
  data: UserStatisticsDto;
  message: string;
}

File: UserManagement/UserManagement.Application/DTOs/UserStatisticsDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace UserManagement.Application.DTOs
{
    public class UserStatisticsDto
    {
        public StatusStatisticsDto StatusStats { get; set; }
        public RoleStatisticsDto RoleStats { get; set; }
    }

    public class StatusStatisticsDto
    {
        public int Active { get; set; }
        public int Inactive { get; set; }
        public double ActivePercentage { get; set; }
        public double InactivePercentage { get; set; }
    }

    public class RoleStatisticsDto
    {
        public int Admin { get; set; }
        public int User { get; set; }
        public double AdminPercentage { get; set; }
        public double UserPercentage { get; set; }
    }
}